flake.nix

```diff
@@ -150,5 +150,9 @@
           };
         };
       };
+
+      # homeModules is preferred but this ensures compatibility for anyone
+      # currently using nix.homeManagerModules.default in their config
+      homeManagerModules.default = self.homeModules.default;
```

Comment:
Were you able to test this to make sure it behaves how you'd expect?

Reply:
Yes, I ran the test locally with `modules = [ nix.homeManagerModules.default ]` instead of `modules = [ nix.homeModules.default ]`